### ISSUE TITLE
[Enhancement]: For New (experimental) product editor

### ISSUE THREAD

**Issue by eliquame** [woothemes/woocommerce#36856]
### Describe the solution you'd like

I'm testin in 7.4 ver Woocommerce new experimental product editor.
Good, light, better than the old version.
I have some recommendation:

- Now, the "div class="product-form-layout"  has max width 1032px. It is funny narrow not only in my 4K monitor, or FullHD laptop. If i change it max-width: inherit it will be GREAT!
- The product descriptions dont have any 1, 2, 3 or more columns option. It is a huge mistake. :-(
- The text moduls (heading and paragraphs) dont have any options, like text color or size.
- Dont like the new editor the ACF attributes

**Comment by samueljseay** [comment]
@joshuatf looks like some feedback here for your team to consider?

**Comment by eliquame** [comment]
@samueljseay Sorry, if I post to wrong place. (It is one of my first github uses in woocommerce side. :-D )

**Comment by samueljseay** [comment]
@eliquame Your feedback is welcomed! I have pinged the teams responsible for the editor :)

**Comment by joshuatf** [comment]
Thank you for the feedback, @eliquame!

>Now, the "div class="product-form-layout" has max width 1032px. It is funny narrow not only in my 4K monitor, or FullHD laptop. If i change it max-width: inherit it will be GREAT!

Could you provide a screenshot of this?

>The product descriptions dont have any 1, 2, 3 or more columns option. It is a huge mistake. :-(

This is referring to the Gutenberg column blocks, right?

>The text moduls (heading and paragraphs) dont have any options, like text color or size.

Great point!

>Dont like the new editor the ACF attributes

Could you expand a bit on what you don't like?

cc @jarekmorawski

**Comment by eliquame** [comment]
Hi @joshuatf 

1.) I will try presentation for you. Here is two screenshot
https://nimb.ws/8Ghtre
https://nimb.ws/DVCM4n

2.)
Yes! I think that a short or long description does not need as many options as a Gutenberg page, but a basic set would be perfectly fine.

- Example contents can be arranged in 1, 2, 3 or 4 columns.
- Image module
- Heading module (color, size, thickness, etc. with stylization)
- Paragraph module (color, size, thickness, etc. with stylization)
- List module (color, size, thickness, etc. with stylization)
- File module. For many products, it is customary to include a pdf document as additional material (e.g. product safety data sheet).
- Divider module
- Image gallery module

I think the implementation of these modules from Gutenberg is more than enough to create a perfect description of content and design.

3.) I think ACF problem only was my mistake. It looks like I was the only one who made a mistake while testing. But I need more time to safely give feedback.

**Comment by pmcpinto** [comment]
Hi @eliquame thank you very much for your feedback. In case you don't mind, I have a couple of follow-up questions:

- How did you find out about the feature? Was it directly in the settings, or did you read about it somewhere?
- Are you using the feature to create/manage real products or are you just trying it out in a test store?
- Are you using extensions that add/manipulate fields in the current product creation/management experience?
- Given what you've seen so far, what are the main advantages of this experimental feature compared to the current product creation/management experience?

Your feedback is more than welcome. As mentioned, this feature is in an experimental phase, so it will evolve a lot in the coming months.

**Comment by eliquame** [comment]
> Hi @eliquame thank you very much for your feedback. In case you don't mind, I have a couple of follow-up questions:
> 
> * How did you find out about the feature? Was it directly in the settings, or did you read about it somewhere?
> * Are you using the feature to create/manage real products or are you just trying it out in a test store?
> * Are you using extensions that add/manipulate fields in the current product creation/management experience?
> * Given what you've seen so far, what are the main advantages of this experimental feature compared to the current product creation/management experience?
> 
> Your feedback is more than welcome. As mentioned, this feature is in an experimental phase, so it will evolve a lot in the coming months.

Hi @pmcpinto !

1.)
First I found it in version 7.3 code. :-) I was just looking for a solution for something else. :-)
Then it was officially released in the Experimental condition in version 7.4. (Function can be switched off)

2.)
Of course, I just test a non-public Developer Site.

3.)
I do not really understand this question within the present framework.

4.)
You have only benefits! (they would be)
After so many years, it could finally give a minimal design to the descriptions without HTML knowledge and/or any responsive problems. It has an incredible deficiency for years. Of course, we always trick the Gutenberg editor using php snippet, but let's face it, it can collapse at any time if the woocommerce is updated (see now)

+1
Didn't I get so deep that this is a completely unique thing, or did you simply bring Gutenberg's own functions? If these are factory gutenberg blocks, you dont have to do nothing only activate the blocks I mentioned above and you're done. There is no other thing to do, no longer experimenting. :-D

**Comment by pmcpinto** [comment]
Hey @eliquame thanks for your feedback.

> 3.)
> I do not really understand this question within the present framework.

Sorry if my question wasn't clear. I mainly wanted to know if you were using extensions like product add-ons or bundles, as this experimental feature still doesn't support extensibility.

**Comment by eliquame** [comment]
Hi @pmcpinto!

 Actually, I wrote that I can't interpret the question "within the current stage", since this is only an experimental function :-) On the other hand, I also only tried it on a subdomain, specially reserved for this kind of game.

Of course, there are several plugins installed on this page, but I haven't dived into it so much as to what happens if I combine the needs with this and this plugin. I simply turned on the experimental feature, and I tried out how the new product editor compares to the traditional tinycme.
